lldb/tools/lldb-dap/tool/CMakeLists.txt

```
@@ -0,0 +1,28 @@
+if(APPLE)
+  configure_file(
+    ${CMAKE_CURRENT_SOURCE_DIR}/lldb-dap-Info.plist.in
+    ${CMAKE_CURRENT_BINARY_DIR}/lldb-dap-Info.plist
+    )
+  # Inline info plist in binary (use target_link_options for this as soon as CMake 3.13 is available)
+  set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -Wl,-sectcreate,__TEXT,__info_plist,${CMAKE_CURRENT_BINARY_DIR}/lldb-dap-Info.plist")
+endif()
+
+add_lldb_tool(lldb-dap
+  lldb-dap.cpp
+
+  LINK_LIBS
+    lldbDAP
+  )
+
```

```suggestion
add_lldb_tool(lldb-dap
  lldb-dap.cpp

  LINK_LIBS
    lldbDAP
  )
  
if(APPLE)
  configure_file(
          ${CMAKE_CURRENT_SOURCE_DIR}/lldb-dap-Info.plist.in
          ${CMAKE_CURRENT_BINARY_DIR}/lldb-dap-Info.plist
  )
  target_link_options(lldb-dap PRIVATE LINKER:-sectcreate,__TEXT,__info_plist,${CMAKE_CURRENT_BINARY_DIR}/lldb-dap-Info.plist)
endif()
```

use [target_link_option](https://cmake.org/cmake/help/latest/prop_tgt/LINK_OPTIONS.html#prop_tgt:LINK_OPTIONS) .

---

I'm just moving the code, but since we're on CMake 3.20 now is as good a time as any to address the fixme.